File: admin/src/main/java/top/zzf4/blog/entity/dto/ArticleUpdateDTO.java
package top.zzf4.blog.entity.dto;

import lombok.Data;

import java.util.List;

@Data
public class ArticleUpdateDTO {

    private Long id;

    private String title;

    private String content;

    private Integer categoryId;

    private Integer wordNum;

    private List<String> tags;

    private String thumbnailUrl;
}

File: blog-backend/internal/dto/article_update_dto.go
package dto

type ArticleUpdateDTO struct {
	ID           int64    `json:"id"`
	Title        string   `json:"title"`
	Content      string   `json:"content"`
	CategoryId   int      `json:"category_id"`
	WordNum      int      `json:"word_num"`
	Tags         []string `json:"tags"`
	ThumbnailUrl string   `json:"thumbnail_url"`
	IsLocked     bool     `json:"is_locked"`
	LockPassword string   `json:"lock_password"`
}
